### PR TITLE
aws: upgrade to CLIv2

### PR DESCRIPTION
**Pull request by codypenta**
Add additional capabilities that came along with V2 that allow for more real-world operations to include: SSO integrations, wizards and other auto helpers, and new output formats.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by zdroid** [comment]
The things causing Travis errors are minor and hard to catch, so I made the changes needed to solve them myself. I also changed `V2` to `v2` since that's the usual way versions are writtten, and also replaced infitive with imperative in the examples.

**Comment by zdroid** [comment]
Note: this shouldn't be merged until @codypenta confirms my changes are alright.

**Comment by codypenta** [comment]
@zdroid LGTM.

File: pages/common/aws.md [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
+`aws {{command}} help`
 
-- Publish message to the specific SNS topic:
+- Get the caller identity (used to troubleshoot permissions):
 
-`aws sns publish --topic-arn {{arn:aws:sns:us-east-1:54633:testTopic}} --message "Message"`
+`aws sts get-caller-identity`
 
-- To see help text for the AWS command:
+- List AWS resources in a region and output in yaml:
```
Comment (agnivade) [inline pages/common/aws.md]: in yaml _format_

File: pages/common/aws.md [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
-`aws {{command}} help`
+`aws dynamodb list-tables --region {{us-east-1}} --output yaml`
+
+- Use auto prompt to help with a command:
+
+`aws iam create-user --cli-auto-prompt`
+
+- Get an interactive wizard for an AWS resource:
+
+`aws dynamodb wizard {{new-table}}`
```
Comment (agnivade) [inline pages/common/aws.md]: new_table

Comment (sbrl) [reply pages/common/aws.md]:
```suggestion
`aws dynamodb wizard {{new_table}}`
```

**Pull request merged**